src/changelog.ts

```diff
@@ -192,42 +187,137 @@ function stringifyLinkReferenceDefinitions(
   //
   // If there have not been any releases yet, the repo URL is used directly as
   // the link definition.
-  const unreleasedLinkReferenceDefinition = `[${unreleased}]: ${
-    hasReleases
-      ? getCompareUrl(repoUrl, `${tagPrefix}${latestSemverVersion}`, 'HEAD')
-      : withTrailingSlash(repoUrl)
-  }`;
+  const unreleasedLinkReferenceDefinition =
+    getUnreleasedLinkReferenceDefinition(
+      repoUrl,
+      tagPrefix,
+      releases,
+      versionBeforePkgRename,
+      tagPrefixBeforePkgRename,
+    );
 
   // The "previous" release that should be used for comparison is not always
```

Comment:
This comment now feels out of place. The code that refers to the "'previous' release" is now inside of `getReleaseLinkReferenceDefinitions`. Should this comment be located there instead? (Perhaps inside the function, at the top, before everything.)